tokio/src/net/driver/reactor/mod.rs

```diff
@@ -321,7 +330,7 @@ impl Handle {
     /// return immediately.
     fn wakeup(&self) {
         if let Some(inner) = self.inner() {
-            inner.wakeup.set_readiness(mio::Ready::readable()).unwrap();
+            inner.waker.wake().expect("failed to wake reactor")
```

Comment:
Don't you want to return the error here?

Reply:
`wakeup` is only used in `Handle::unpark` which does not return anything. I think handling an error here could be considered, but not necessarily as part of this change. Similar to before, if there an unexpected reason why `wakeup` fails then a panic will happen.